copt/randomized.py

```diff
@@ -691,3 +693,165 @@ def epoch_iteration_template(
             memory_gradient[i] = grad_i
 
     return epoch_iteration_template
+
+
+def step_size_sfw(variant):
+    if variant in {'SAG', 'SAGA'} :
+        def step_sizes_SAG_A(t, n_samples=None):
+            step_size_x = 2 / (t+2)
+            return step_size_x, None
+        return step_sizes_SAG_A
+
+    if variant == 'MK':
```

Comment:
Prefer "moktari" or something more. At the very least, define this somewhere in the file.